src/NUnitFramework/framework/Constraints/NUnitEqualityComparer.cs

```diff
@@ -85,6 +85,11 @@ internal NUnitEqualityComparer()
             };
         }
 
+        internal NUnitEqualityComparer(NUnitEqualityComparer copy)
```

Comment:
I can't figure out where this is used.

Reply:
It's not, looks like I forgot to remove it.